services/activities/activity_definitions/schemas.go

```diff
@@ -168,6 +168,11 @@ var argumentMappingSchema = map[string]*schema.Schema{
 		Type:     schema.TypeString,
 		Required: true,
 	},
+	"mapping_status": {
+		Type:        schema.TypeString,
+		Computed:    true,
```

Comment:
should we have a ValidateFunc, even if it is computed field?

Reply:
I don't think it's necessary. ValidateFunc is used for input fields to validate the value provided by the user

Reply:
good for me